lib/presentation/screens/maps/maps.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:convert';
+import 'package:btd6wiki/models/map.dart';
+import 'package:btd6wiki/presentation/screens/maps/single_map.dart';
+import 'package:btd6wiki/utilities/global_state.dart';
+import 'package:btd6wiki/utilities/images_url.dart';
+import 'package:flutter/material.dart';
+import 'dart:async';
+import 'package:flutter/services.dart' show rootBundle;
+
+class Maps extends StatefulWidget {
+  const Maps({required Key key}) : super(key: key);
+
+  @override
+  _MapsState createState() => _MapsState();
+}
+
+class _MapsState extends State<Maps> {
+  List<MapModel> _jsonData = [];
+
+  @override
+  void initState() {
+    super.initState();
+    _loadJsonData();
+  }
+
+  Future<void> _loadJsonData() async {
```

Comment:
Will this be in a loop for all the maps?

Reply:
No, need to make a json with the items and when entering a map load it